examples/listing-articles.html

```diff
@@ -0,0 +1,196 @@
+								<li class="checkbox">
+									<label>
+										<input type="checkbox" name="article_type" class="wb-tagfilter-ctrl" value="type__media_advisory"> Media Advisory
+									</label>
+								</li>
+								<li class="checkbox">
+									<label>
+										<input type="checkbox" name="article_type" class="wb-tagfilter-ctrl" value="type__news_conference"> News Conference
+									</label>
+								</li>
+								<li class="checkbox">
+									<label>
+										<input type="checkbox" name="article_type" class="wb-tagfilter-ctrl" value="type__news_release"> News Release
+									</label>
+								</li>
+							</ul>
+						</fieldset>
+					</div>
+					<hr>
+					<!-- Topic -->
+					<div class="form-group mrgn-bttm-0">
+						<label for="topicList">Topic</label>
+						<select id="topicList" name="topic" class="form-control full-width wb-tagfilter-ctrl">
+							<option value="">All</option>
+							<option value="topic__housing">Housing</option>
+							<option value="topic__human_rights">Human Rights</option>
+							<option value="topic__income_security">Income Security</option>
+							<option value="topic__information_management">Information Management</option>
+						</select>
+					</div>
+					<hr>
+					<div class="form-group mrgn-bttm-0">
+						<label for="releaseYear">Release Year</label>
+						<select id="releaseYear" name="releaseYear" class="form-control full-width wb-tagfilter-ctrl">
+							<option value="">All</option>
+							<option value="year__2025">2025</option>
+							<option value="year__2024">2024</option>
+							<option value="year__2023">2023</option>
+							<option value="year__2022">2022</option>
+							<option value="year__2021">2021</option>
+						</select>
+					</div>
+				</details>
+			</div>
+
+			<!-- Reports listing -->
+			<div class="col-xs-12 col-md-8 col-lg-9">
+				<div class="d-flex mb-2">
+					<p class="wb-fltr-info ml-auto">
+						<span class="text-muted"><span data-nbitem></span> Results</span>
+					</p>
+				</div>
+				<div class="row wb-eqht-grd hght-inhrt reports-list wb-tagfilter-items" data-wb-json='{
+						"url": "articles_en.json#/events",
+						"mapping": [
+							
+							{ "selector": ".listing-card", "attr": "data-wb-tags", "value": "/tags" },
+							{ "selector": ".article-type", "value": "/type" },
+							{ "selector": "time", "value": "/date" },
+							{ "selector": "time", "attr": "datetime", "value": "/date" }
+						]
+					}'>
+					<template>
+						<div class="col-xs-12 col-md-6 listing-card" data-wb-tags="">
+							<article class="oag-card bg-light h-100">
+								<div class="mb-3">
+									<span class="label label-default article-type"></span>
+									<time datetime="" class="small"><span></span></time>
+								</div>
+								<p><a href="article.html" class="stretched-link mb-2"></a>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</a></p>
+							</article>
```

Comment:
Title with `<h3>` instead of `<p>` and add the class `oag-heading-as-body` to the card